### PR TITLE
Make OSS Page Dynamic

### PR DESCRIPTION
**Pull request by HarryET**
Use Octokit Core to dynamically fetch info about Supabase org and update the OSS page

## What kind of change does this PR introduce?

Improves the OSS page of the documentation

## What is the current behavior?

Static out of date data is used. See below:
image

## What is the new behavior?

All content - that can be done without authentication - is dynamically fetched from the GitHub API

## Additional context

https://canary.discord.com/channels/839993398554656828/867348150741237790/921521989485723668

### CODE REVIEW

File: web/src/pages/oss.js [hunk 1]
```diff
@@ -90,7 +102,10 @@ export default function Oss() {
         <div className="container">
           <h2>Repositories</h2>
           <div className="row is-multiline">
-            {repos.map((props, idx) => (
+            {repos.length < 1 && <div>
+              {/* TODO Add Spinner Here */}
```
Comment (HarryET) [inline web/src/pages/oss.js:106]: How would a loader or spinner be implemented? Or is there a better/easier solution?

File: web/sidebar_spec_cli.js [hunk 2]
```diff
@@ -3,13 +3,19 @@ module.exports = {
...
       label: 'About',
-      items: ['reference/cli/index', 'reference/cli/supabase-init'],
+      items: ['reference/cli/index'],
       collapsed: true,
     },
     {
       type: 'category',
-      label: 'Local Development',
-      items: ['reference/cli/supabase-start', 'reference/cli/supabase-stop', 'reference/cli/supabase-eject'],
+      label: 'Command reference',
```
Comment (MildTomato) [inline web/sidebar_spec_cli.js:11]: is this branch out of date from master?

Comment (HarryET) [reply web/sidebar_spec_cli.js:11]: Was up to date with master last night when I did it. I can see if I need to merge master back in though?

**Comment by florian-lefebvre** [comment]
Is there any way to preview the new page ?

File: web/sidebar_spec_cli.js (hunk 2 above)
Comment (HarryET) [reply web/sidebar_spec_cli.js:11]: I should have merged master into my branch now

File: web/src/pages/oss.js (hunk 1 above)
Comment (MildTomato) [reply web/src/pages/oss.js:106]: Not sure to be honest with docusaurus

**Comment by MildTomato** [comment]
sorry for delay on this, i'll check it out later - looks good though

**Comment by HarryET** [comment]
> sorry for delay on this, i'll check it out later - looks good though

@MildTomato any chance you could enable deploys with vercel?

**Comment by HarryET** [comment]
Changed the page to only show 6, as per @Isaiah-Hamilton commented on in a DM
Screenshot 2022-02-17 at 21 40 49

**Review by Isaiah-Hamilton — APPROVED** [review]
lgtm

File: web/src/pages/oss.js [hunk 3]
```diff
@@ -18,12 +21,24 @@ export default function Oss() {
     .filter((v, i, a) => a.indexOf(v) === i) // remove duplicates
     .sort((a, b) => a.localeCompare(b)) // alphabetical
   const maintainerPills = ['All'].concat(maintainerTags)
-  
+
+  React.useEffect(async () => {
+    const reposResponse = await octokit.request("GET /orgs/{org}/repos", {
+      org: "supabase",
+      type: "public",
+      sort: "updated",
```
Comment (HarryET) [inline web/src/pages/oss.js:29]: Surely this wants to be the six biggest libs, or have it be paginated.

Comment (kiwicopple) [reply web/src/pages/oss.js:29]: Let's just make it all of them. Keep it simple for now

Comment (Isaiah-Hamilton) [reply web/src/pages/oss.js:29]: Sounds good, will do so in the updated [pull request](https://github.com/supabase/supabase/pull/5693).

**Comment by Isaiah-Hamilton** [comment]
Closing this pull request because it cant be merged due to the conflicting file `package-lock.json`, will use the updated [pull request](https://github.com/supabase/supabase/pull/5693) instead.

**Pull request closed without merging**